test/test_helper_discovery.rb

```diff
@@ -108,10 +110,18 @@ def organization_one
   Organization.find_by_name('Organization 1')
 end
 
+def organization_two
+  Organization.find_by_name('Organization 2')
+end
+
 def location_one
   Location.find_by_name('Location 1')
 end
 
+def location_two
+  Location.find_by_name('Location 2')
+end
+
 def current_path_info
   current_url.sub(%r{.*?://}, '')[%r{[/\?\#].*}] || '/'
 end
```

Comment:
Hmm this will slow down all tests, I'd consider putting this only into the test where you need it. Or make it lazy-loaded.